autogluon/inference/docker/1.2/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,71 @@
+ARG PYTHON_VERSION=3.11.9
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:2.5.1-cpu-py311-ubuntu22.04-sagemaker
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get autoremove -y \
+ && apt-get install tesseract-ocr -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG AUTOGLUON_VERSION=1.2.0
+
+# Upgrading pip and installing/updating Python dependencies
+# Comments are added to explain the reason behind each update
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U wheel \
+ && pip uninstall -y dataclasses \
+ && pip install --no-cache-dir -U numpy numba \
+# Install AutoGluon, ensuring no vulnerable dependencies are left behind
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+# Update requests to a secure version that fixes CVE-2023-32681
+ && pip install --no-cache-dir -U requests>=2.31.0 \  
+# Capping setuptools to 69.5.1 to fix AutoMM tests
+ && pip install --no-cache-dir setuptools==69.5.1 \  
+ # Update pillow to a secure version that fixes CVE-2023-50447
+ && pip install --no-cache-dir -U pillow>=10.2.0 \
+ # Update Jinja2 to a secure version that fixes vulnerability id 64227
+ && pip install --no-cache-dir -U Jinja2>=3.1.3 \
+ # Update aiohttp to a secure version that fixes vulnerability id 64642
+ && pip install --no-cache-dir -U aiohttp>=3.9.2 \
+ # update tqdm to a secure version that fixes CVE-2024-34062
+ && pip install --no-cache-dir -U tqdm>=4.66.3 \
+ # update urllib3 to fix vulnerability id 71608
+ && pip install --no-cache-dir -U urllib3 \
+  # Cap pillow & ninja to fix sanity test
+ && pip install --no-cache-dir "pillow<11.0.0" \
+ && pip install --no-cache-dir "ninja<1.11.1.1"
```

Comment:
Can you validate all these overrides are still needed for 1.2.0 release? Thanks

Reply:
removed the unnecessary ones